torchao/quantization/transform_module.py

```diff
@@ -47,5 +47,6 @@ def _transform(
     @functools.wraps(config_type)
     def decorator(func):
         _QUANTIZE_CONFIG_HANDLER[config_type] = func
+        return func  # needed to make the functions usable externally
```

Comment:
?

Reply:
if you wanted to use any of the transform functions in quant_api.py that use the decorator, it would say that the function is actually just `None` since nothing was being returned after the function was passed into the decorator. They were only stored in the transform function dict so they were only usable through quantize_. This solves that problem. Since for MoE we use these transform functions, this was necessary and doesn't affect anything.